tests/phpunit/tests/xmlrpc/wp/newPost.php

```diff
@@ -259,19 +305,19 @@ function test_terms_names() {
 		$this->make_user_by_role( 'editor' );
 
 		$ambiguous_name = 'foo';
-		$parent_cat = wp_create_category( $ambiguous_name );
-		$child_cat = wp_create_category( $ambiguous_name, $parent_cat );
+		$parent_cat     = wp_create_category( $ambiguous_name );
+		$child_cat      = wp_create_category( $ambiguous_name, $parent_cat );
 
 		$cat1_name = 'cat1';
-		$cat1 = wp_create_category( $cat1_name, $parent_cat );
+		$cat1      = wp_create_category( $cat1_name, $parent_cat );
 		$cat2_name = 'cat2';
 
 		// first a post with valid categories; one that already exists and one to be created
-		$post = array(
-			'post_title' => 'Test',
+		$post   = array(
+			'post_title'  => 'Test',
 			'terms_names' => array(
-				'category' => array( $cat1_name, $cat2_name )
-			)
+				'category' => array( $cat1_name, $cat2_name ),
+			),
 		);
 		$result = $this->myxmlrpcserver->wp_newPost( array( 1, 'editor', 'editor', $post ) );
```

Comment:
Should items in this array start on new lines since it has more than one item?

Reply:
It's not an associative array.